.changes/unreleased/operator-Added-20250115-162158.yaml

```diff
@@ -0,0 +1,20 @@
+project: operator
+kind: Added
+body: |
+  Added `resources.limits` and `resources.requests` as an alternative method of managing the redpanda container's resources.
+
+    When both `resources.limits` and `resources.requests` are specified, the
+    redpanda container's `resources` will be set to the provided values and all
+    other keys of `resources` will be ignored. Instead, all other values will be
+    inferred from the limits and requests.
+
+    This allows fine grain control of resources. i.e. It is now possible to set
+    CPU requests without setting limits:
+
+    ```yaml
+    resources:
+      limits: {} # Specified but no cpu or memory values provided
+      requests:
+        cpu: 5 # Only CPU requests
+    ```
+time: 2025-01-15T16:21:58.898733-05:00
```

Comment:
Why operator release notes are the copy of the redpanda helm chart release notes?

Reply:
Because this change is also relevant to the operator now that it's CRDs have been updated. IMO it's an implementation detail that the operator is just the chart. Users shouldn't have to go to the charts change log to see relevant changes to the operator. WDYT?

Reply:
ok